pkgs/by-name/ss/sss_code/package.nix

```
@@ -0,0 +1,49 @@
+{
+  rustPlatform,
+  fetchFromGitHub,
+  pkg-config,
+  fontconfig,
+  libiconv,
+  stdenv,
+  libxcb,
+  lib,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "sss_code";
+  version = "0.2.0";
+
+  src = fetchFromGitHub {
+    owner = "SergioRibera";
+    repo = "sss";
+    rev = "sss_code/v${version}";
+    hash = "sha256-AmJFAwHfG4R2iRz9zNeZsVFLptVy499ozQ7jgwnevOo=";
+  };
+
+  useFetchCargoVendor = true;
+
+  cargoHash = "sha256-qeDZgrGPSz+wXolZeVb2FFHjLzl1+vjzMN/3NCgaf/s=";
+
+  # Removed 'sss' command due to version differences and to separate commands.
+  postInstall = ''
+    rm -f $out/bin/sss
+  '';
+
+  nativeBuildInputs = [ pkg-config ] ++ lib.optionals stdenv.buildPlatform.isDarwin [ libiconv ];
+
+  buildInputs = [
+    fontconfig
+    libxcb
+  ];
+
```

```suggestion

  doCheck = false; # no tests

```
The tests take quite some time to compile otherwise.

---

@FliegendeWurst  Okay, I'll change it.